pages/instructorQuestionManualGrading/instructorQuestionManualGradingNextInstanceQuestion.sql

```diff
@@ -0,0 +1,18 @@
+-- BLOCK get_next_unmarked_instance_question
+SELECT iq.*
+FROM 
+    instance_questions AS iq
+        JOIN assessment_questions AS aq ON (aq.id = iq.assessment_question_id)
+        JOIN assessments AS a ON (a.id = aq.assessment_id)
+        JOIN variants AS v ON (v.instance_question_id = iq.id)
+        JOIN (
+            -- We only want the LAST submissions that are created by each user to join on all
+            -- instance questions under assessment question
+            SELECT DISTINCT ON (s.auth_user_id) * FROM submissions AS s
```

Comment:
This query won't work for group work, where multiple users might be submitting. In general, we should use `auth_user_id` for anything other than auditing. In this case, we want the most recent submission per instance question, not per `auth_user_id`.

I would actually prefer to use the instance question status to select them and ignore submissions entirely.